riscv/processor.cc

```diff
@@ -741,30 +743,11 @@ const char* processor_t::get_privilege_string()
   abort();
 }
 
-void processor_t::set_virt(bool virt)
-{
-  reg_t tmp, mask;
-
-  if (state.prv == PRV_M)
-    return;
-
-  /*
-    * Ideally, we should flush TLB here but we don't need it because
-    * set_virt() is always used in conjucter with set_privilege() and
-    * set_privilege() will flush TLB unconditionally.
-    *
-    * The virtualized sstatus register also relies on this TLB flush,
-    * since changing V might change sstatus.MXR and sstatus.SUM.
-    */
-  state.prev_v = state.v;
-  state.v = virt;
-}
-
 void processor_t::enter_debug_mode(uint8_t cause)
 {
   state.debug_mode = true;
   state.dcsr->write_cause_and_prv(cause, state.prv, state.v);
-  set_privilege(PRV_M);
+  set_privilege(PRV_M, false);
```

Comment:
This looks like a functional change, since it wasn't forcing V=0 previously?

Reply:
I guess so, but it was definitely a bug before, because prv=M and V=1 is paradoxical.